com.endlessm.CompanionAppService.json.in

```diff
@@ -24,6 +24,27 @@
     ],
     "modules": [
         "com.endlessm.CompanionAppService.PipDependencies.json",
+        {
+            "name": "mustache-c",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/x86-64/mustache-c"
+                }
+            ]
+        },
+        {
+            "name": "meos-knowledge-content-renderer",
+            "buildsystem": "meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/endlessm/eos-knowledge-content-renderer",
+                    "branch": "T21272"
```

Comment:
Quick note here that we'll need to remove this branch once eknr is merged.

Reply:
Indeed.